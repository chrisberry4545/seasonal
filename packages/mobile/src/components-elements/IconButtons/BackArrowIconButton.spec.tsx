import React from 'react';
import { BackArrowIconButton } from './BackArrowIconButton';
import { shallow, ShallowWrapper } from 'enzyme';

jest.mock('../Buttons', () => ({
  BareButton: () => 'BareButton'
}));
jest.mock('../Icons', () => ({
  ArrowIcon: () => 'ArrowIcon'
}));

describe('<BackArrowIconButton />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(
      <BackArrowIconButton
        onClick={() => undefined}
        activeOpacity={0.3}
        style={{ color: 'blue' }} />
    );
  });

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());
});
