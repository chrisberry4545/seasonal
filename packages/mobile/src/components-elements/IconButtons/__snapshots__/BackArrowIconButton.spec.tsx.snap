// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackArrowIconButton /> renders correctly 1`] = `
<BareButton
  activeOpacity={0.3}
  onClick={[Function]}
  style={
    Object {
      "color": "blue",
    }
  }
>
  <ArrowIcon
    size={24}
  />
</BareButton>
`;
